src/utils/nexmark/src/utils.rs

```diff
@@ -83,9 +81,14 @@ pub fn get_base_url(seed: u64) -> String {
     )
 }
 
-pub fn build_channel_url_map(channel_number: usize) -> HashMap<usize, (String, String)> {
-    let mut ans = HashMap::new();
-    ans.reserve(channel_number);
+lazy_static::lazy_static! {
```

Comment:
Use `LazyLock` here?

Reply:
It is not stable yet so better avoided in a public crate. 🥲